src/code/views/inspector-panel-view.coffee

```diff
@@ -2,31 +2,67 @@ NodeInspectorView = React.createFactory require './node-inspector-view'
 LinkInspectorView = React.createFactory require './link-inspector-view'
 PaletteInspectorView = React.createFactory require './palette-inspector-view'
 
-{div, i} = React.DOM
+{div, i, span} = React.DOM
 
-module.exports = React.createClass
+ToolButton = React.createFactory React.createClass
+  displayName: 'toolButton'
+  render: ->
+    name = @props.name
+    onClick = =>
+      @props.onClick(@props.name) if @props.onClick
```

Comment:
You can also write this as

    @props.onClick? @props.name

it "compiles" to

    var base;
    if (typeof (base = this.props).onClick === "function") {
      base.onClick(this.props.name);
    }